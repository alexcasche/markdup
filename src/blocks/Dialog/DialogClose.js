import { Button } from 'elements'
import { tint } from 'polished'

const DialogClose = Button.extend`
  background-color: ${props => tint(0.92, props.theme.colors.black)};
  position: absolute;
  top: 0;
  right: 0;
  padding: 8px 11px 8px 12px;
  border-radius: 0 5px 0 5px;
  font-size: 16px;
  svg {
    color: ${props => tint(0.65, props.theme.colors.grey)};
  }
  &:hover {
    background-color: ${props => tint(0.94, props.theme.colors.black)};
    svg {
      color: ${props => tint(0.6, props.theme.colors.grey)};
    }
  }
`

export default DialogClose
