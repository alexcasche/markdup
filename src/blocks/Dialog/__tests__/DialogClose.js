import DialogClose from '../DialogClose'

describe('DialogClose Element', () => {
  it('renders correctly', () => {
    const tree = render(<DialogClose />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
