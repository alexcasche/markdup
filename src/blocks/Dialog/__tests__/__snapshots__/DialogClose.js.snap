// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogClose Element renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border: none;
  outline: none;
  font-family: "IBM Plex Sans Condensed",Helvetica,Arial,sans-serif;
  font-weight: 500;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  text-transform: uppercase;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 0.15s ease-in-out;
  transition: all 0.15s ease-in-out;
  background-color: #2f3032;
  position: absolute;
  top: 0;
  right: 0;
  padding: 8px 11px 8px 12px;
  border-radius: 0 5px 0 5px;
  font-size: 16px;
}

.c0:hover {
  cursor: pointer;
}

.c0 svg {
  color: #c8c9cd;
}

.c0:hover {
  background-color: #2a2c2e;
}

.c0:hover svg {
  color: #cccdd1;
}

<button
  className="c0"
/>
`;
